packages/ida.plugin.scyllahide.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,28 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+
+    $installDir    = Join-Path ${Env:ProgramFiles} "ScyllaHide"
```

Comment:
```suggestion
    $toolName = "ScyllaHide"
    $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
```

Tools should be installed to `${Env:RAW_TOOLS_DIR}` unless it is not possible to do this easily (but it is not the case here). Please also use variable names consistent with other packages. There is [documentation in the Wiki](https://github.com/mandiant/VM-Packages/wiki/Coding-Conventions#common-variable-names). If we only install it as plugin, we can use names consistent with `ida.plugin.capa.vm`.

Do we want to install ScyllaHide as tool?
- If so, I suggest we use the `VM-Install-From-Zip` helper and the binary `ScyllaTest` as executable for the shortcut link. 
- If not, I think we should download it to a temporary directory (`$tempDownloadDir = Join-Path ${Env:chocolateyPackageFolder} "temp_$([guid]::NewGuid())"`) and not keep the rest of the files.

Reply:
Resolved at https://github.com/mandiant/VM-Packages/pull/701/commits/31c0229c121684d5c7e18ab82a0a87aa135edbcd.